jjb/edgex-templates-snap.yaml

```diff
@@ -368,3 +368,33 @@
       - lf-provide-maven-settings-cleanup
       - edgex-provide-snap-cleanup
       - edgex-provide-docker-cleanup
+- job-template:
+    name: '{project-name}-release-snap'
+
+    # Job template for Snap release job
+    #
+    # The purpose of this job template is to run snapcraft release
+
+    <<: *snap_job_boiler_plate
+    # yamllint disable-line rule:key-duplicates
+    <<: *snap_merge_boiler_plate
+
+    parameters:
+      - string:
+          name: SNAP_CHANNEL
+          description: 'Snap channel to release to.'
+      - string:
+          name: SNAP_REVISION
+          description: 'Snap revision to release.'
+
+    builders:
+      - lf-jacoco-nojava-workaround
+      - lf-provide-maven-settings:
```

Comment:
These settings files provide docker credentials. I don't see them being used anywhere, and the job seems to run fine on the sandbox without them. I would suggest removing lines 392-394 and 398.

Reply:
The snap job runs inside a docker container but doesn't release anything to dockerhub, nexus, etc. so if these credentials are for releasing things like that, then these credentials can be removed.

Reply:
Yes, they are for Nexus repos and dockerhub.

Reply:
Understood, I removed the `lf-provide-maven-settings` and `lf-provide-maven-settings-cleanup` from this job template.